packages/bpk-component-content-cards/src/ContentCard/ContentCard.module.scss

```
@@ -0,0 +1,91 @@
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable selector-max-compound-selectors */
+@import '~bpk-mixins/index';
+
+$bpk-spacing-v2: true;
+
+@mixin aspect-ratio($width, $height) {
+  position: relative;
+
+  &::before {
+    content: '';
+    display: block;
+    width: 100%;
+    padding-top: ($height / $width) * 100%;
+  }
+
+  > :first-child {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+  }
+}
+
+.bpk-content-cards {
+  &__cardLink {
+    color: inherit;
+    font: inherit;
+    text-decoration: inherit;
+  }
+
+  &__cardHorizontal {
+    display: grid;
+    column-gap: bpk-spacing-xxl();
+    grid-template-columns: 50% 40%;
+    align-items: center;
+
+    @include bpk-breakpoint-mobile {
+      grid-template-columns: 1fr;
+    }
+  }
+
+  &__cardHorizontal &__imageAspectRatio {
```

I think nested classes have to be updated to `kebab-case` also, like this:
```suggestion
  &__card-horizontal &__image-aspect-ratio {
```

---

Just make sure this follows BEM, as this is the format Backpack uses